tests/org.eclipse.reddeer.eclipse.test/META-INF/MANIFEST.MF

```diff
@@ -22,4 +22,5 @@ Require-Bundle: org.eclipse.reddeer.go;bundle-version="[2.0,2.1)",
  org.eclipse.epp.logging.aeri.ide
 Bundle-ActivationPolicy: lazy
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
-Import-Package: org.hamcrest.core
+Import-Package: org.hamcrest.core,
+ org.jboss.reddeer.swt.impl.menu
```

Comment:
org.eclipse.reddeer.go is already in Require-Bundle so no need for reddeer.swt in Import-Package